tests/test_tokenize_v3.py

```diff
@@ -205,15 +323,37 @@ def test_tool_message_multiple_shots_with_history(tokenizer: InstructTokenizer)
         )
     )
     _, text = tokenized.tokens, tokenized.text
-    assert text == (
-        "<s>[INST]▁a[/INST]"
-        '[TOOL_CALLS]▁[{"name":▁"b",▁"arguments":▁{},▁"id":▁"0"}]</s>[TOOL_RESULTS]▁{"content":▁"d",▁"call_id":▁"0"}[/TOOL_RESULTS]'
-        "▁e</s>[INST]▁f[/INST]"
-        '[TOOL_CALLS]▁[{"name":▁"b",▁"arguments":▁{},▁"id":▁"1"}]</s>[TOOL_RESULTS]▁{"content":▁"d",▁"call_id":▁"1"}[/TOOL_RESULTS]'
-    )
+    assert text == expected_text
 
 
-def test_tool_message_multiple_calls(tokenizer: InstructTokenizer) -> None:
+@pytest.mark.parametrize(
+    "tokenizer, expected_text",
+    [
+        (
+            tokenizer(),
+            "<s>[INST]▁a[/INST]"
+            '[TOOL_CALLS]▁[{"name":▁"b",▁"arguments":▁{},▁"id":▁"0"},▁{"name":▁"q",▁"arguments":▁{},▁"id":▁"1"}]</s>'
+            '[TOOL_RESULTS]▁{"content":▁"d",▁"call_id":▁"0"}[/TOOL_RESULTS]'
+            '[TOOL_RESULTS]▁{"content":▁"d",▁"call_id":▁"1"}[/TOOL_RESULTS]'
+            "▁e</s>[INST]▁f[/INST]"
+            '[TOOL_CALLS]▁[{"name":▁"b",▁"arguments":▁{},▁"id":▁"2"},▁{"name":▁"q",▁"arguments":▁{},▁"id":▁"3"}]</s>'
+            '[TOOL_RESULTS]▁{"content":▁"d",▁"call_id":▁"2"}[/TOOL_RESULTS]'
+            '[TOOL_RESULTS]▁{"content":▁"d",▁"call_id":▁"3"}[/TOOL_RESULTS]',
```

Comment:
(nit) could we maybe define these templates above:

```py
TEST_TOOL_TEMPLATES = ...

```

it could make it easier to read and maintain